File: Dec2020/11Dec/RepeatedKTimes.java
/*
Problem Name: Repeated K Times
Problem Link: https://www.hackerearth.com/practice/algorithms/searching/linear-search/practice-problems/algorithm/repeated-k-times/
*/
import java.io.*;
import java.util.*;
class TestClass {
    public static void main(String args[] ) throws Exception {
        Scanner in = new Scanner(System.in);
        PrintWriter out = new PrintWriter(System.out);
        int n = in.nextInt(), k;
        int map[] = new int[100001];
        for(int i=0;i<n;i++)
            map[in.nextInt()]++;
        k = in.nextInt();
        for(int i=1;i<100001;i++)
            if(map[i] == k){
                System.out.print(i);
                break;
            }
        out.flush();
    }
}

File: Dec2020/11Dec/RepeatedKTimes.cpp
/*
Problem Name: Repeated K Times
Problem Link: https://www.hackerearth.com/practice/algorithms/searching/linear-search/practice-problems/algorithm/repeated-k-times/
*/

#include<bits/stdc++.h>
using namespace std;

int main(){
    ios::sync_with_stdio(0);    
    cin.tie(0);             
    int n,k;
    cin>>n;
    int A[n];
    map<int,int> m;
    for(int i=0;i<n;++i){
        cin>>A[i];
        m[A[i]]++;
    }
    cin>>k;
    for(auto l: m){
        if(l.second == k){
            cout<<l.first;
            break;
        }
    }
    return 0;
}
